### ISSUE TITLE
Further development

### ISSUE THREAD

**Issue by kaiec** [dmcoles/EstyJs#8]
Dear Darren,

besides the changes in screen rendering (as shown in #5), I now also changed the mouse positioning (mouse cursors are now usually synced, a resync can also be triggered via middle mouse button).

I also changed the TOS version to EmuTos, making the whole project fully open source without any potential copyright issues.

I plan to look into the sound issues (#2). 

My actual goal is to make this an easy to use package to publish ST games as browser games, e.g. on itch.io. It is almost there, an additional nice feature would be the support of a full screen mode with proper scaling.

As you stated that you won't develop EstyJS further, I would like to know your opinion how I should proceed, mainly regarding the name EstyJS. I see three options:

1. You are ok with me developing this as EstyJS under https://github.com/kaiec/EstyJS. I would suggest to call it EstyJS 2 to distinguish it from your version and a link to my repo in your README and website would be great to help people find the new version. Of course I would link to your repo as well and credit you appropriately, as I do already.
2. You want to be part of this, then I suggest to create an EstyJS organization on GitHub and to transfer the codebase there so that we both can maintain it (and potentially others in the future when I also will have lost my interest)
3. You do not want me to take over the development of EstyJS and just keep it as it is. Then I would rename my fork to something else so that people are not confused. A link from your repo and website would of course also be appreciated and credits to you will be there, so basically this is option 1 with a new name.

I prefer option 1 (assuming you do not want to develop this further), as I like the name EstyJS and think it would be nice to keep it alive this way.

Let me know what you think!

Cheers,

Kai

**Comment by dmcoles** [comment]
Hi.
Thanks for the email. I am pleased you feel that you wish to take this project forwards.
I am absolutely happy for you to continue development from your own repo as version 2. I simply don't have time these days to maintain this as i have other projects that take up my time. I will happily put a link to your repo in the readme and a link to your version on the website.

Darren

    On Tuesday, 12 November 2024 at 08:52:29 GMT, Kai Eckert ***@***.***> wrote:  
 
 


Dear Darren,

besides the changes in screen rendering (as shown in #5), I now also changed the mouse positioning (mouse cursors are now usually synced, a resync can also be triggered via middle mouse button).

I also changed the TOS version to EmuTos, making the whole project fully open source without any potential copyright issues.

I plan to look into the sound issues (#2).

My actual goal is to make this an easy to use package to publish ST games as browser games, e.g. on itch.io. It is almost there, an additional nice feature would be the support of a full screen mode with proper scaling.

As you stated that you won't develop EstyJS further, I would like to know your opinion how I should proceed, mainly regarding the name EstyJS. I see three options:
   
   - You are ok with me developing this as EstyJS under https://github.com/kaiec/EstyJS. I would suggest to call it EstyJS 2 to distinguish it from your version and a link to my repo in your README and website would be great to help people find the new version. Of course I would link to your repo as well and credit you appropriately, as I do already.
   - You want to be part of this, then I suggest to create an EstyJS organization on GitHub and to transfer the codebase there so that we both can maintain it (and potentially others in the future when I also will have lost my interest)
   - You do not want me to take over the development of EstyJS and just keep it as it is. Then I would rename my fork to something else so that people are not confused. A link from your repo and website would of course also be appreciated and credits to you will be there, so basically this is option 1 with a new name.

I prefer option 1 (assuming you do not want to develop this further), as I like the name EstyJS and think it would be nice to keep it alive this way.

Let me know what you think!

Cheers,

Kai
